source/cps/index.rst

```diff
@@ -0,0 +1,29 @@
+CPS
+===
+
+:软件名称: CPS
+:主页: http://www.eas.slu.edu/eqc/eqccps.html
+:源码地址: http://www.eas.slu.edu/eqc/eqc_cps/CPS/cpslisc.html
+:官方文档: http://www.eas.slu.edu/eqc/eqc_cps/CPS/CPS330.html
+:作者: `Saint Louis University Earthquake Center <http://www.eas.slu.edu/eqc/eqc.html>`__
+:最新版本: v3.30 (2020-10-29)
```

Comment:
The version is tricky here. CPS v3.30 was released long time ago. They're fixing bugs but never bump the package version. 

Perhaps we should add a warning or note in the file.

Reply:
done